src/runtime_src/core/pcie/driver/linux/xocl/subdev/clock.c

```diff
@@ -654,6 +497,21 @@ static inline int clock_wiz_busy(struct clock *clock, int idx, int cycle,
 	return 0;
 }
 
+static inline unsigned int floor_acap_o(int freq)
+{
+	return (4320 / freq);
+}
```

Comment:
Since we are removing the comment about the python script, does it make sense to comment or (better, if possible) predefine some macro about 4320, 1000, 333333?

Reply:
For 4320:
There is a full document: https://confluence.xilinx.com/display/XIP/VCK5000+kernel+clocking. I don't know how reliable to put confluence into the code, possible shouldn't do that. 

I think 1000 with 33.333 is good enough. I tested 10000 with 333333, rolled it back.

Reply:
> For 4320:
> There is a full document: https://confluence.xilinx.com/display/XIP/VCK5000+kernel+clocking. I don't know how reliable to put confluence into the code, possible shouldn't do that.
> 
> I think 1000 with 33.333 is good enough. I tested 10000 with 333333, rolled it back.

definitely not the conf page. I am just suggesting a macro to explain what is 4320, something like 
#define VERSAL_CLK_MAX_VCO 4320
#define VERSAL_CLK_INPUT_FEQ 33.333